cmd/eval_test.go

```diff
@@ -514,8 +514,7 @@ package test
 # METADATA
 # schemas:
 #   - input: schema["input"]
-p { 
-	rego.metadata.rule() # presence of rego.metadata.* calls must not trigger unwanted schema evaluation 
```

Comment:
Yeah, I removed because I thought that unit test was testing the type mismatch when using inlined schema vs not which is done by the lines:
`- input: schema["input"]` 
and 
`input.foo == 42 # type mismatch with schema that should be ignored`

so that `rego.metadata.rule()` was not necessary,  but now that you explained to me I see that we had 2 tests being done together. I will put it back.